zeebe.js

```diff
@@ -0,0 +1,97 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE
+};
+
+// An action that assigns a user task.
```

Comment:
A bit of a nit pick since we are in a transition period with user tasks. Do we need to call this a "Zeebe user task" or is this sufficient?

Reply:
Agreed here, always best to be too detailed 😄